ParabolInc/parabol
# feat: Create embeddings for meeting templates

# Description

Fixes #9772 
[Please include a summary of the changes and the related issue]

## Demo

[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]

## Testing scenarios

- run the migration
- start the embedder
- wait for a bit
- see the embeddings being generated in PG

## Final checklist
